angularjs-portal-home/src/main/webapp/my-app/layout/widget/controllers.js

```diff
@@ -4,6 +4,9 @@ define(['angular'], function(angular){
 
   var app = angular.module('my-app.layout.widget.controllers', []);
 
+  app.controller('ListOfLinksController', ['$scope', 'layoutService', function($scope, layoutService){
+  }]);
```

Comment:
Right. Kind of put it in there as a placeholder but we could definitely take it out til it's necessary.